admin-management-service/src/main/java/com/scaler/adminmanagementservice/controllers/AdminController.java

```diff
@@ -0,0 +1,54 @@
+package com.scaler.adminmanagementservice.controllers;
+
+import com.scaler.adminmanagementservice.dtos.GenericAdminDto;
+import com.scaler.adminmanagementservice.services.AdminService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/admins")
+public class AdminController implements AdminOperations{
+
+    private AdminService adminService;
+
+    public AdminController(AdminService adminService){
+
+        this.adminService = adminService;
+    }
+
+
+
+    @GetMapping("{id}")
```

Comment:
/id/{id},
I don't think this code is tested. Not sure how you are hitting endpoints like this